modules/gitleaks/Makefile

```diff
@@ -0,0 +1,10 @@
+export GITLEAKS_VERSION ?= 1.2.0
+
+GITLEAKS ?= gitleaks
+
+## Install gitleaks
+gitleaks/install: packages/install/gitleaks
+
+## Scan current repository
+gitleaks/scan:
```

Comment:
should this call `gitleaks/scan: gitleaks/install`?

Reply:
i don't think so, because modules like `helm` or `chamber` also have install target, but didn't invoke it in other targets